site/documentation/static/api/device-registry-v1.yaml

```diff
@@ -486,6 +486,10 @@ components:
                description: The Base64 encoded binary DER encoding of the
                             trusted root certificate. Either this property
                             or `public-key` must be set.
+            "algorithm":
+               type: string
+               description: The algorithm of the key material.
+               example: RSA
```

Comment:
RSA or EC

Reply:
Unfortunately multiple examples are still unsupported by the swagger UI: https://github.com/swagger-api/swagger-ui/issues/2651

I will use "RSA" as default and "EC" as example.